axelrod/strategies/shortmem.py

```diff
@@ -0,0 +1,36 @@
+from axelrod import Actions, Player
+from axelrod.actions import Action
+
+C, D = Actions.C, Actions.D
+
+class ShortMem(Player):
+    """A player who only ever defects."""
+
+    name = 'ShortMem'
+    classifier = {
+        'memory_depth': 10,
+        'stochastic': False,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    @staticmethod
+    def strategy(opponent: Player) -> Action:
+        
+        memoryDepth = self.classifier['memoryDepth']
+        cooperateRatio = array.count('C')/memoryDepth
```

Comment:
You haven't defined `array` so I expect this strategy would fail to play.

Note that we're going to need some tests for the strategy too, you can see some information on the tests here: http://axelrod.readthedocs.io/en/latest/tutorials/contributing/strategy/writing_test_for_the_new_strategy.html

We're happy to give a hand with tests if you'd like one. Essentially we want code that checks that all the logic you write below works :)

Also, note that a `Player` has a `cooperations` attribute that counts the number of cooperations.